shared/files/agent/montagu-docker-login

```diff
@@ -11,16 +11,22 @@ vault auth -method=github > /dev/null
 
 REGISTRY_URL=docker.montagu.dide.ic.ac.uk:5000
 REGISTRY_USER=vimc
-HUB_USER=vimcrobot
 REGISTRY_PASSWORD=$(vault read -field=password /secret/registry/${REGISTRY_USER})
 
+HUB_USER=vimcrobot
+HUB_PASSWORD=$(vault read -field=password /secret/${HUB_USER$)
```

Comment:
Syntax error here I think. Also, we seem to have both a folder and a secret at `secret/vimcrobot` in the vault?

Reply:
Thanks, have deleted the folder